### ISSUE TITLE
Filter upcoming board by label

### ISSUE THREAD

**Issue by Dubidubiduu** [nextcloud/deck#2398]
**Is your feature request related to a problem? Please describe.**
Upcoming board is messy. I do have finished tasks (which still stands there for documentation), and also my important/middle-important/not-important cards flying around in one stack without order

**Describe the solution you'd like**
1.) A filter-setting for labels, similar to what we have for boards, this way I can filter for important/not-important.
2.) But then I would still get those finished. So either I archive finished cards or #534 is needed here.

**Describe alternatives you've considered**
Don't know, maybe swim-lane #32 could be an idea. So it could be filtered at least by board.

Later then I still think #2232 is something to consider.

**Comment by juliushaertl** [comment]
So, how is it different from https://github.com/nextcloud/deck/issues/2398 (which is still open)? :grin:

**Comment by Dubidubiduu** [comment]
> 
> 
> So, how is it different from #2398 (which is still open)? 😁

Guess you meant #2232, as #2398 is this issue itself. ;)

And surely, you have a point. Need to put more effort in describing things.

For me, #2232 is (now) more about the idea to also filter/tailer for other users, while this is just tags. But I can merge them if you want me to do.

**Comment by juliushaertl** [comment]
Yes, I'd say having a generic filter interface makes sense here. We can by default just show the current user ones and then allow the user to filter by label or extend to other users.
